pinot-tools/src/main/java/org/apache/pinot/tools/MultistageEngineQuickStart.java

```diff
@@ -94,7 +94,6 @@ public String[] getDefaultBatchTableDirectories() {
   @Override
   public Map<String, Object> getConfigOverrides() {
     Map<String, Object> overrides = new HashMap<>(super.getConfigOverrides());
-    overrides.put("pinot.multistage.engine.enabled", "true");
     overrides.put("pinot.server.instance.currentDataTableVersion", 4);
```

Comment:
By default data table version is 3. I think enabling multistage engine alone wouldn't be sufficient since queries will fail with data-table v3.

https://github.com/apache/pinot/blob/29fd5d9cd947f936c52bd768d45ab52088abdb30/pinot-core/src/main/java/org/apache/pinot/core/common/datatable/DataTableBuilderFactory.java#L37

Reply:
Good point, any idea this doesn't fail on integration test?

Reply:
cc: @walterddr

Reply:
datatable version is set to 4 in multistage integration tests